### PR TITLE
[CLB] Implement Endless Evil

### PR DESCRIPTION
**Pull request by PurpleCrowbar**
Implements [[Endless Evil]]. My first time trying to implement a card but it all seems to work as expected.

### CODE REVIEW

**Comment by PurpleCrowbar** [comment]
The [card in question](https://scryfall.com/card/clb/665/endless-evil) since the card fetcher bot doesn't seem to work for PRs:
image
On a side note, even though I can test this card by using cheats on a test server, I can't see it in the card viewer or in the deck editor. Is there something additional that must be done to have it appear in these?

**Comment by Alex-Vasile** [comment]
> The [card in question](https://scryfall.com/card/clb/665/endless-evil) since the card fetcher bot doesn't seem to work for PRs:
> image
> On a side note, even though I can test this card by using cheats on a test server, I can't see it in the card viewer or in the deck editor. Is there something additional that must be done to have it appear in these?

when you add a card you have to delete the card database from the client and server. (either manually or by doing a clean build)

**Comment by PurpleCrowbar** [comment]
> > The [card in question](https://scryfall.com/card/clb/665/endless-evil) since the card fetcher bot doesn't seem to work for PRs:
> > On a side note, even though I can test this card by using cheats on a test server, I can't see it in the card viewer or in the deck editor. Is there something additional that must be done to have it appear in these?
> 
> when you add a card you have to delete the card database from the client and server. (either manually or by doing a clean build)

Thank you, that was it. Have now also added the borderless variant of the card while I was at it.

**Comment by Alex-Vasile** [comment]
> > > The [card in question](https://scryfall.com/card/clb/665/endless-evil) since the card fetcher bot doesn't seem to work for PRs:
> > > On a side note, even though I can test this card by using cheats on a test server, I can't see it in the card viewer or in the deck editor. Is there something additional that must be done to have it appear in these?
> > 
> > 
> > when you add a card you have to delete the card database from the client and server. (either manually or by doing a clean build)
> 
> Thank you, that was it. Have now also added the borderless variant of the card while I was at it.

See my review above. Other than that it looks good

**Comment by PurpleCrowbar** [comment]
> > > > The [card in question](https://scryfall.com/card/clb/665/endless-evil) since the card fetcher bot doesn't seem to work for PRs:
> > > > On a side note, even though I can test this card by using cheats on a test server, I can't see it in the card viewer or in the deck editor. Is there something additional that must be done to have it appear in these?
> > > 
> > > 
> > > when you add a card you have to delete the card database from the client and server. (either manually or by doing a clean build)
> > 
> > 
> > Thank you, that was it. Have now also added the borderless variant of the card while I was at it.
> 
> See my review above. Other than that it looks good

I can't seem to find the review in question?

File: Mage.Sets/src/mage/cards/e/EndlessEvil.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                        false, null, 1, 1, false);
+                effect.setTargetPointer(new FixedTarget(enchantment.getAttachedTo(), game));
+                return effect.apply(game, source);
+            }
+        }
+        return false;
+    }
+}
+
+class EndlessEvilBounceEffect extends TriggeredAbilityImpl {
```
[the remark this answers is not in the document]
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EndlessEvil.java:102]: This

File: Mage.Sets/src/mage/cards/e/EndlessEvil.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        return new EndlessEvilBounceEffect(this);
+    }
+
+    @Override
+    public boolean checkEventType(GameEvent event, Game game) {
+        return event.getType() == GameEvent.EventType.ZONE_CHANGE;
+    }
+
+    @Override
+    public boolean checkTrigger(GameEvent event, Game game) {
```
[the remark this answers is not in the document]
Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/e/EndlessEvil.java:123]: And this

**Review by Alex-Vasile — CHANGES_REQUESTED** [review]

**Comment by Alex-Vasile** [comment]
Sorry, realized I left them as pending by mistake

**Comment by PurpleCrowbar** [comment]
Should be good to go now

**Pull request merged**